docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -119,7 +119,7 @@ A draft implementation is optional, but you may find it useful to create one bef
 ### Resources
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31962&commitId=45f4dc989ea593030ab7bca1f8ab30c8a747ecd7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>